src/Bitshares/Memo.h

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2017-2019 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include <string>
+#include <nlohmann/json.hpp>
+
+#include "../PublicKey.h"
+#include "../PrivateKey.h"
+#include "../Data.h"
+
+namespace TW::Bitshares {
+
+// Encrypts message using AES-256 in CBC mode and PKCS#5 padding
+// The key must be 256 bits long and the IV 128 bits.
+TW::Data aesEncrypt(const uint8_t *message, size_t messageLength, const uint8_t *key, const uint8_t *initializationVector);
```

Comment:
can we move this into a standardalone C++ wrapper?

Reply:
Sure. The function is totally independent of the bitshares code. But it's not flexible. There are no provisions for specifying the keylength, padding scheme, etc.

Reply:
That's fine, you can create it under Bitshares folder